pkg/common/common.go

```diff
@@ -199,6 +203,8 @@ type ProvisionerConfiguration struct {
 	// LabelsForPV could be used to specify additional labels that will be
 	// added to PVs created by static provisioner.
 	LabelsForPV map[string]string `json:"labelsForPV" yaml:"labelsForPV"`
+	// SetPVOwnerRef indicates if PVs should be dependents of the owner Node, default to false
+	SetPVOwnerRef bool `json:"setPVOwnerRef" yaml:"setPVOwnerRef"`
```

Comment:
Can we add this option to the helm chart (default off) too?

Reply:
Sure, done here: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/123/commits/74869f0c8ecc273c543618c1d37892aebb1dfbd3